README.md

```diff
@@ -15,3 +15,29 @@ during this experimentation phase.
 We don't want to discourage others from trying to port stuff if it helps us learn how to work in lean 4,
 but please understand that anything that is currently in mathlib4 is subject to change/deletion,
 and the "real" port hasn't started yet
+
+# Build instructions
+
+* Get the newest version of `elan`. If you already have installed a version of Lean, you can run
+  ```
+  elan self update
+  ```
+  If the above command fails, or if you need to install `elan`, run
+  ```
+  curl https://raw.githubusercontent.com/leanprover/elan/master/elan-init.sh -sSf | sh
+  ```
+  If this also fails, follow the instructions under `Regular install` [here](https://leanprover-community.github.io/get_started.html).
+* Install `lake`
+  ```
+  cd .. # go to a folder where you want to clone repositories
+  git clone https://github.com/leanprover/lake.git
+  cd lake
+  lake +Kha/lean4:v4.0.0-bundle-rc build
+  ./build/bin/lake # this should output some usage information
+  cd ../mathlib4
+  ```
```

Comment:
```suggestion
```

You should never install lake manually.  Lake is bundled with lean, and the bundled lake version is the only supported one to use.

Reply:
Ah, thanks. I didn't realize that the lake I used was bundled with Lean. My mistake!